Packs/Core/TestPlaybooks/Test_Playbook_-_Impossible_Traveler_Response.yml

```diff
@@ -0,0 +1,2689 @@
+      AbuseIPDBThreshold:
+        simple: "80"
+      AutoContainment:
+        simple: "False"
+      CommentToAdd:
+        simple: '${alert.name}. Alert ID: ${alert.id}'
+      ContactUserManager:
+        simple: "False"
+      MaxMilesPerHourAllowed:
+        simple: "400"
+      ShouldOpenTicket:
+        simple: "False"
+      ZendeskSubject:
+        simple: XSIAM Incident ID - ${parentIncidentFields.incident_id}
+      addCommentPerEndpoint:
+        simple: "True"
+      description:
+        simple: ${parentIncidentFields.description}. ${parentIncidentFields.xdr_url}
+      domain:
+        simple: demisto
+      preInvestigationContainment:
+        simple: "False"
+      serviceNowShortDescription:
+        simple: XSIAM Incident ID - ${parentIncidentFields.incident_id}
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "268":
+    id: "268"
+    taskid: 6c73cb45-9d7a-445f-83db-d21f79ac77ec
+    type: regular
+    task:
+      id: 6c73cb45-9d7a-445f-83db-d21f79ac77ec
+      version: -1
+      name: Verify Context Error - Impossible Traveler IPs
+      description: Prints an error entry with a given message
+      scriptName: PrintErrorEntry
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      message:
+        simple: |-
+          The 'ImpossibleTravelerIPs' context key was not extracted properly. One of the following may be the cause:
+          1- The 'CreateArray' script failed to execute.
+          2- The 'contextKey' input configuration was changed for the 'CreateArray' automation used in the 'Get all Impossible Traveler IP Addresses' task.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1540,
+          "y": 1415
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "269":
+    id: "269"
+    taskid: ac870eec-4348-41c3-8900-d31277a80df6
+    type: condition
+    task:
+      id: ac870eec-4348-41c3-8900-d31277a80df6
+      version: -1
+      name: Verify Block Indicators
+      description: Verify that the ‘BlockIndicators’ context key was extracted correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "270"
+      "yes":
+      - "256"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: BlockIndicators
```

Comment:
Does this key will be created with the provided IPs? I double-checked them with VT and seems like they r not malicious... maybe we should mark one of them as malicious to get this key?